test/unit/tui_spec.lua

```diff
@@ -153,6 +153,50 @@ itp('handle_background_color', function()
   rbuffer.rbuffer_consumed(rbuf, #term_response)
 
 
+  -- Handles background color with the OSC command part split.
+  term_response = '\027]11;rgb'
```

Comment:
#25918 moves TUI bg response handling to Lua, and deletes this test file.

Reply:
Thanks. I've rebased this PR, with the following updates:

- Removed the `tui_spec.lua` tests.
- Updated  `parse_osc` for compatibility with the changes to `libvterm`.
- Changed the expected color of "Press ENTER or type command to continue" for one of the tests.